parser.y

```diff
@@ -7726,6 +7727,18 @@ SetStmt:
 		}
 		$$ = &ast.SetStmt{Variables: assigns}
 	}
+|	"SET" "CONFIG" "TIDB" VariableName EqOrAssignmentEq SetExpr
+	{
+		$$ = &ast.SetConfigStmt{Type: "TiDB", Name: $4, Value: $6}
+	}
+|	"SET" "CONFIG" identifier VariableName EqOrAssignmentEq SetExpr
+	{
+		$$ = &ast.SetConfigStmt{Type: $3, Name: $4, Value: $6}
+	}
```

Comment:
Perhaps better use a single rule with `Identifier` and store the type in lower-case like

```suggestion
|	"SET" "CONFIG" Identifier VariableName EqOrAssignmentEq SetExpr
	{
		$$ = &ast.SetConfigStmt{Type: strings.ToLower($3), Name: $4, Value: $6}
	}
```